docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -15,8 +15,15 @@ type                = "directrequest"
 schemaVersion       = 1
 name                = "example eth request event spec"
 contractAddress     = "0x613a38AC1659769640aaE063C651F48E0250454C"
-# Optional externalJobID: Automatically generated if unspecified
-externalJobID       = "0EEC7E1D-D0D2-476C-A1A8-72DFB6633F02"
+
+# Optional fields:
+# requesters        = [
+#   "0xaaaa1F8ee20f5565510B84f9353F1E333E753B7a",
+#   "0xbbbb70F0e81C6F3430dfdC9fa02fB22BdD818C4e"
+# ]
+# minContractPaymentLinkJuels = "100000000000000"
```

Comment:
I tested this on a 1.3.0 node. When I add `minContractPaymentLinkJuels` without the `#` character, it doesn't show up in the job definition. It behaves the same way as any other invalid field where the field is completely ignored and doesn't show up in the final job spec definition when you view it in the operators UI. Add `invalidField = "10101"`, for example. Are we sure this field is actually functional?

The `requesters` field works fine and shows up in the definition as expected.

Reply:
PR addressing this issue: https://github.com/smartcontractkit/chainlink/pull/6487

Reply:
@dwightjl Fixed on develop, and it was only a UI limitation so we can probably merge these docs now :shrug: